2-mandatory/7-eligible-students.js

```diff
@@ -14,7 +14,15 @@
   - Returns an array containing only the names of the who have attended AT LEAST 8 classes
  */
 
-function eligibleStudents(attendances) {}
+function eligibleStudents(attendances) {
+  let eligibleNames = [];
+  attendances.forEach((student) => {
+    if (student.attendance >= 8) {
+      eligibleNames.push(student.name);
+    }
+  });
+  return eligibleNames;
+}
 
```

Comment:
Nice work , well done Doris, Also you can also using filter array method to save some keystorkes. 👍